lib/aws/generated/evidently.ex

```diff
@@ -565,12 +573,20 @@ defmodule AWS.Evidently do
         project,
         max_results \\ nil,
         next_token \\ nil,
+        status \\ nil,
```

Comment:
Oh, this chance the function signature by adding a new param :/
I think we need to review those services and introduce the usage of `Keyword` lists for these optional params.